sklearn/linear_model/stochastic_gradient.py

```diff
@@ -1290,8 +1242,6 @@ def _fit_regressor(self, X, y, alpha, C, loss, learning_rate,
         # Windows
         seed = random_state.randint(0, np.iinfo(np.int32).max)
 
-        tol = self._tol if self._tol is not None else -np.inf
```

Comment:
maybe this line should stay here, making None an alias for ``-np.inf``?

Reply:
I agree.
In any case, we have to be consistent, either keeping this line, or removing the one in `fit_binary`:

https://github.com/scikit-learn/scikit-learn/blob/92a50e5748f55868333bd51d3b5c86482245d9b0/sklearn/linear_model/stochastic_gradient.py#L389